sdk/src/cameras/itof-camera/camera_itof.cpp

```diff
@@ -734,6 +734,12 @@ aditof::Status CameraItof::requestFrame(aditof::Frame *frame) {
         return status;
     }
 
+    if (m_dropFirstFrame) {
+        m_depthSensor->getFrame(frameDataLocation);
+        m_dropFirstFrame = false;
+        LOG(INFO) << "Dropped first frame";
+    }
```

Comment:
This isn't the proper solution. What you are doing here is disabling the setting to drop the 1st frame.
Think about when the application will close, the JSON will be set with the setting set to off.
Variable m_dropFirstFrame should not be touched unless it is desired to change the setting.